src/components/footer/footer.component.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import footerLogo from '../../assets/footer-logo.svg';
+import fbLogo from '../../assets/f_logo_white.svg';
+import twitterLogo from '../../assets/twitter_logo_white.svg';
+
+import './footer.styles.scss';
+
+const scrollToTop = () => {
+    window.scrollTo(0, 0);
+};
+
+const Footer = () => (
+    <div className="footer-background">
+        <div className="footer-top-section">
+
+            <div className="footer-left-side">
+                <div className="footer-logo-group">
+                    <img src={footerLogo} alt="footer logo" />
+                    <div className="footer-smaller-logos">
+                        <img src={fbLogo} alt="footer logo" />
+                        <img src={twitterLogo} alt="footer logo" />
+                    </div>
+                </div>
+            </div>
+
+            <div className="footer-middle">
+                <div className="page-info-group-container">
+                    <div className="page-info-group">
+                        <p className="page-name">CONCEPT</p>
+                        <p className="page-content">DEAR COCOとは</p>
+                        <p className="page-content">ココナッツの力</p>
+                        <p className="page-content">ブランドストーリー</p>
+                    </div>
+
+                    <div className="page-info-group">
+                        <p className="page-name">QUALITY</p>
+                        <p className="page-content">フェアトレードについて</p>
+                        <p className="page-content">主な成分</p>
+                        <p className="page-content">SDGsへの取り組み</p>
+                    </div>
+
+                    <div className="page-info-group">
+                        <p className="page-name">PRODUCTS</p>
+                        <p className="page-content">商品一覧</p>
+                        <p className="page-content">新商品ピックアップ</p>
+                    </div>
+
+                    <div className="page-info-group">
+                        <p className="page-name">SHOPS</p>
+                        <p className="page-content">取扱店舗一覧</p>
+                        <p className="page-content">オンラインショップ</p>
+                        <p className="page-content">ショップガイド</p>
+                    </div>
+
+                    <div className="page-info-group">
+                        <p className="page-name">NEWS</p>
+                        <p className="page-content">新着情報</p>
+                        <p className="page-content">メディア掲載実績</p>
+                    </div>
+
+                    <div className="page-info-group">
+                        <p className="page-name">CONTACT</p>
+                        <p className="page-content">FAQ - よくある質問</p>
+                        <p className="page-content">問い合わせフォーム</p>
```

Comment:
all of these should be links. in another task you'll have them auto scroll to their respective section when clicked.
the design doesn't show how these links look when interacted with so you can get creative.
try adding styles for all the [link states](https://www.bitdegree.org/learn/css-link-color#link-states)

Reply:
added some styling